marketplaceItems/partyBall/entityResources/entityServerScripts/partyBall_server.js

```diff
@@ -295,7 +295,16 @@
 
     // Clear the explodeTimer if there is one
     function unload() {
-        explodeTimer && Script.clearInterval(explodeTimer);
+        if (explodeTimer) {
+            Script.clearInterval(explodeTimer);
+            Music.stop();
```

Comment:
should all of these functions get executed on unload even if `explodeTimer` is falsey?

Reply:
hmm, I don't think so because the counterpart to those functions are only invoked to create entities if there is an explodeTimer created.